File: app/src/main/java/org/ole/planet/myplanet/utilities/JsonParserUtils.java
package org.ole.planet.myplanet.utilities;

import com.google.gson.JsonArray;
import com.google.gson.JsonElement;
import com.google.gson.JsonParser;

public class JsonParserUtils {
    public static JsonArray getStringAsJsonArray(String s) {
        JsonParser parser = new JsonParser();
        JsonElement arrayElement = parser.parse(s);
        return arrayElement.getAsJsonArray();
    }
}

File: app/src/main/java/org/ole/planet/myplanet/utilities/JsonParserUtils.kt
package org.ole.planet.myplanet.utilities

import com.google.gson.JsonArray
import com.google.gson.JsonParser.parseString

object JsonParserUtils {
    fun getStringAsJsonArray(s: String?): JsonArray {
        val arrayElement = parseString(s)
        return arrayElement.asJsonArray
    }
}
